packages/next/build/webpack/loaders/next-flight-server-loader.ts

```diff
@@ -116,18 +165,14 @@ async function parseModuleInfo({
             }
           }
         } else {
-          // For the client compilation, we skip all modules imports but
-          // always keep client/shared components in the bundle. All client components
-          // have to be imported from either server or client components.
           if (
-            isServerComponent(importSource) ||
-            hasFlightLoader(importSource, 'server') ||
-            // TODO: support handling RSC components from node_modules
-            isNodeModuleImport
+            !addClientImport(importSource, {
```

Comment:
We know it will return fase for `isNodeModuleImport` and `isBuiltinModule` before passing them in. 
```js
if (isNodeModuleImport || isBuiltinModule) continue
addClientImport(importSouce)
```